demo/index.html

```diff
@@ -8,8 +8,7 @@
       content="JavaScript Polyfill for the Sanitizer API"
     />
     <title>Demo for the Sanitizer API Polyfill</title>
-    <script src="../vendor/purify.min.js"></script>
-    <script src="../src/polyfill.js" type="module"></script>
+    <script src="../dist/sanitizer-polyfill.js"></script>
```

Comment:
Do we need to make it so that this will require some additional build step for githug pages to work?
I kinda like that we get instant testing of the `main` branch in https://mozilla.github.io/sanitizer-polyfill/demo/

Reply:
I don't expect (nor do I want) contributors to always submit a change to `dist/` when they change a file.. I wonder if we can make that work on GitHub Pages with GitHub Actions. Maybe this should be a follow-up. What's your take?

(P.S: Sorry, this PR has been sitting so long. This project is unfortunately not my highest priority at the moment).

Reply:
It can be done with Github Actions. I went ahead and added the workflow `build-demo.yml` that builds the dist files and commits them when new commits are pushed to the `main` branch. The important part is the `[skip actions]` in the message of the new commit. It tells github to not run workflows for that commit. Otherwise it would get stuck in an infinite commit-workflow loop.